### PR TITLE
hide voltage and deadband fields in ratio

### PR DESCRIPTION
**Pull request by anistouri**

### CODE REVIEW

File: src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js [hunk 1]
```diff
@@ -46,6 +46,9 @@ const RatioTapChangerPane = ({
     isModification = false,
 }) => {
     const { trigger } = useFormContext();
+    const watchLRegulationMode = useWatch({
```
Comment (flomillot) [inline src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js:49]: It's already present below, remove it
And you can remove the associated comment

File: src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js [hunk 2]
```diff
@@ -271,19 +274,23 @@ const RatioTapChangerPane = ({
                 <Grid item container xs={4}>
                     {ratioTapLoadTapChangingCapabilitiesField}
                 </Grid>
-                <Grid item container spacing={2}>
-                    {isRatioTapLoadTapChangingCapabilitiesOn && (
+                {isRatioTapLoadTapChangingCapabilitiesOn && (
```
Comment (flomillot) [inline src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js:277]: Can you refactor it please ? The condition with `isRatioTapLoadTapChangingCapabilitiesOn` is already present 3 times below.

**Review by flomillot — CHANGES_REQUESTED** [review]

File: src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js (hunk 1 above)
Comment (anistouri) [reply src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js]: Done

File: src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js (hunk 2 above)
Comment (anistouri) [reply src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js:272]: Done

File: src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js [hunk 3]
```diff
@@ -275,61 +270,60 @@ const RatioTapChangerPane = ({
...
+                            >
+                                <FormattedMessage
+                                    id="RegulatedTerminal"
+                                    disabled={true}
+                                />
+                            </Grid>
+                            <Grid item xs={4}>
+                                {regulationTypeField}
+                            </Grid>
+                            {isRatioTapLoadTapChangingCapabilitiesOn &&
```
Comment (flomillot) [inline src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js:307]: You can remove this one which is useless

**Review by flomillot — CHANGES_REQUESTED** [review]

**Review by flomillot — CHANGES_REQUESTED** [review]
You can remove all the `!isRatioTapLoadTapChangingCapabilitiesOn` which are useless now

File: src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js (hunk 3 above)
Comment (anistouri) [reply src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js]: Done

**Review by flomillot — APPROVED** [review]

**Review by EstherDarkish — CHANGES_REQUESTED** [review]
If I put "-" (the minus sign) in the deadband field, then hide it by unchecking the on-load mini slider, then I cannot submit the form and the error is not obvious.

**Review by EstherDarkish — CHANGES_REQUESTED** [review]
I can still submit "-" in the deadband field, which causes a HTTP400 error. Unavailable fields should not be sent to the back.

**Review by EstherDarkish — APPROVED** [review]
Test OK Code OK

**Pull request merged**